site/_includes/partials/extensions/reusing-prod-extension-id.md

```diff
@@ -28,11 +35,9 @@ same ID.
 
 ### Compare IDs {: #extension_management }
 
-Open the Extensions Management page at `chrome://extensions`, ensure developer mode is enabled and
-upload the unpackaged extension directory. Compare the extension ID on the extensions management
+Open the Extensions Management page at `chrome://extensions`, ensure **Developer mode** is enabled
+and upload the unpackaged extension directory. Compare the extension ID on the extensions management
 page to the Item ID in the Developer Dashboard. They should match.
 
-{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/AGorME3hdXd2YeKot5Nc.png", 
-alt="The ID of the
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/AGorME3hdXd2YeKot5Nc.png", alt="The ID of the
 extension match", width="356", height="352" %}
```

Comment:
```suggestion
extension match", width="356", height="352", class="screenshot" %}

```

Add the "screenshot" class to surround the image with an outline. Also, the file should end with a blank line. See [this SO answer](https://stackoverflow.com/a/729795/4959520) for more info.